_data/simple-icons.json

```diff
@@ -5655,7 +5655,12 @@
         {
             "title": "OCaml",
             "hex": "EC6813",
-            "source": "http://ocaml.org/img/OCaml_Sticker.svg"
+            "source": "http://ocaml.org/img/OCaml_Sticker.svg",
+            "guidelines": "https://ocaml.org/docs/logos.html",
+            "license": {
+                "type": "CC-BY-SA-4.0",
```

Comment:
The icon is under 'Unlicense' as mentioned in that LICENSE.md file. We can also remove the link as it doesn't differ from the default for that license.

Reply:
Hum... You're right...
Should we accept the type `UNLICENSE` for the license? It would make clear that this icon is unlicensed and not just that we did not find any license to it.

Reply:
In fact, we have!
I will update it :)

Reply:
Think we can also remove the URL - as it looks like the default Unlicense terms mentioned there.

Reply:
Done 😃